web/beacon-app/src/application/routes/paths.ts

```diff
@@ -21,6 +21,10 @@ export const ROUTES = {
   FORGOT_PASSWORD: '/forgot-password',
   RESET_PASSWORD: '/reset-password',
   RESET_VERIFICATION: '/reset-verification',
+  DATA_PLAYGROUND: 'https://rotational.io/data-playground/',
```

Comment:
A link to the Data Playground is listed in `EXTERNAL_LINKS`. We should use that one instead of adding a link here. https://github.com/rotationalio/ensign/blob/f0b67fcd38f318ea30b4d2e5a1bfa360ed365952/web/beacon-app/src/application/routes/paths.ts#L60C28-L60C28

Reply:
@daniellemaxwell Could you please consider creating a 1-point story for the upcoming sprint focusing on centralizing duplicated constants? The top of this file should include a user-friendly comment explaining how to add external links. The external link constant could also be moved to this location. 
Otherwise, I get confused when working because I don't remember whether this file even exists. Ben had the same issue before when trying to update the calendar link.